README.md

```diff
@@ -1 +1,167 @@
-Paginator
+<H1 align="center">Paginator</H1>
+
+<p align="center">
+<a href="https://cocoapods.org/pods/incetro-paginator"><img alt="Version" src="https://img.shields.io/cocoapods/v/incetro-paginator.svg?style=flat"></a> 
+<a href="https://github.com/Incetro/paginator/blob/master/LICENSE"><img alt="Liscence" src="https://img.shields.io/cocoapods/l/incetro-paginator.svg?style=flat"></a> 
+<a href="https://developer.apple.com/"><img alt="Platform" src="https://img.shields.io/badge/platform-iOS-green.svg"/></a> 
+<a href="https://developer.apple.com/swift"><img alt="Swift4.2" src="https://img.shields.io/badge/language-Swift5.0-orange.svg"/></a>
+</p>
+
+## Description
+This library is API responses pagination wrapper.
+
+- [Usage](#Usage)
+- [Requirements](#requirements)
+- [Communication](#communication)
+- [Installation](#installation)
+- [Authors](#license)
+
+## Usage <a name="Usage"></a>
+
+To use a *Paginator*, all you will need to import `Paginator' module into your swift file:
```

Comment:
To use the *Paginator*, all you need is to import `Paginator` module into your swift file: